app/client/src/features/accounts/UserMenu/UserMenu.tsx

```diff
@@ -179,10 +179,20 @@ export function UserMenu({ className, queryRef }: UserMenuProps) {
                     </Button>
                     <ResponsiveDialog open={type === 'login'} onClose={close}>
                         <DialogContent>
-                            <LoginForm onSuccess={() => { close(); router.reload(); }} />
+                            <LoginForm
+                                onSuccess={() => {
+                                    close();
+                                    router.reload();
+                                }}
+                            />
                         </DialogContent>
                     </ResponsiveDialog>
-                    <Button color='primary' variant='outlined' onClick={handleClick('register')}>
+                    <Button
+                        data-test-id='appbar-register-button'
```

Comment:
Does this get filtered out when transpiling to js for prod?

Reply:
I'm not sure, I'd have to check. It was one of the suggestions by playwright but I might have to add something to next's config for it to filter it out.

Reply:
Yeah, it's not a big deal tbh, was more so curious.

Reply:
Mainly because, if there's a way to do it without having all of those artifacts in a production deployment, it's gonna be a pain to clean up once we have more tests.

Reply:
Maybe we can brainstorm more about different strategies for selecting particular buttons/actions.